### PR TITLE
kie-issues#793: The keyboard shortcuts panel doesn't show the new DMN Editor keyboard shortcuts

### LINKED ISSUE #793: The keyboard shortcuts panel doesn't show the new DMN Editor keyboard shortcuts
The keyboard shortcut panel is not showing any keyboard shortcuts from the new DMN Editor.

Chrome
image

Safari
image

Acceptance criteria
- [ ] The keyboard shortcut panel should render all shortcuts from the new DMN Editor.

### PR DESCRIPTION
**Pull request by ljmotta**
Closes: https://github.com/apache/incubator-kie-issues/issues/793

## Description
Move the keyboard shortcuts to the DMN Editor envelope.

https://github.com/apache/incubator-kie-tools/assets/24302289/a9559496-7bf6-4f25-b323-31fa6bbdfdce

### CODE REVIEW

**Review by jomarko — APPROVED** [review]
The current state of the PR seems to work fine also the listed shortcuts! Many thanks ! Some shortcuts are totally new for me!

File: packages/dmn-editor/src/commands/CommandsContextProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    createGroup: () => {},
+    selectAll: () => {},
+    panDown: () => {},
+    panUp: () => {},
+    paste: () => {},
+    copy: () => {},
+    cut: () => {},
+    cancelAction: () => {},
+    focusOnSelection: () => {},
+    resetPosition: () => {},
```
Comment (tiagobento) [inline packages/dmn-editor/src/commands/CommandsContextProvider.tsx]: Maybe the default should be throwing an error? I mean, none of these should ever be a no-op.

**Review by tiagobento — CHANGES_REQUESTED** [review]
Simple change on the default behavior for shortcuts... Not sure if it makes sense, though. I always prefer throwing than silently failing, though. 

Anyway, also please create an issue for tests to be added to each shortcut individually. Thanks @ljmotta. Great implementation!

**Comment by jomarko** [comment]
if there is a ticket/epic for tests, please share a link with us, thank you @ljmotta

**Comment by ljmotta** [comment]
@jomarko @tiagobento I've incorporated the review and I've created the issue https://github.com/apache/incubator-kie-tools/issues/3925

**Review by tiagobento — APPROVED** [review]

**Pull request merged**